File: src/main/java/HelloWorld/src/test/java/HelloWorldTest.java
package HelloWorld.src.test.java;

import HelloWorld.src.main.java.HelloWorld;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class HelloWorldTest {

    @Test
    public void helloNoName() {
        assertEquals("Hello, World!", HelloWorld.hello(""));
        assertEquals("Hello, World!", HelloWorld.hello(null));
    }

    @Test
    public void helloSampleName() {
        assertEquals("Hello, Alice!", HelloWorld.hello("Alice"));
    }

    @Test
    public void helloAnotherSampleName() {
        assertEquals("Hello, Bob!", HelloWorld.hello("Bob"));
    }
}

File: kotlinapp/src/test/kotlin/com/kotlinapp/kotlinintro/HelloWorldTest.kt
package com.kotlinapp.kotlinintro

import hello
import org.junit.Assert.*
import org.junit.Ignore
import org.junit.Test

/**
 * @author lusinabrian on 20/05/17.
 */
class HelloWorldTest {
    @Test
    fun helloNoName() {
        assertEquals("Hello, World!", hello())
    }

    @Test
    fun helloSampleName() {
        assertEquals("Hello, Alice!", hello("Alice"))
    }

    @Test
    fun helloBlankName() {
        assertEquals("Hello, World!", hello("    "))
    }

    @Test
    fun helloAnotherSampleName() {
        assertEquals("Hello, Bob!", hello("Bob"))
    }
}